packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -315,14 +315,15 @@ const Dropdown = ({
     }, [is_list_visible]);
 
     const handleSelect = (item: TListItem) => {
-        if (item.value !== value && onChange) onChange({ target: { name: name || '', value: item.value } });
+        if (item.value !== value) onChange?.({ target: { name: name || '', value: item.value } });
 
         handleVisibility();
     };
 
     const handleVisibility = () => {
         if (typeof onClick === 'function') {
             onClick();
+
             return;
```

Comment:
```suggestion
            onClick?.();
```

Reply:
just a suggestion ...

Reply:
Just for my understanding, why you added optional chain here ? :thinking:

Reply:
there is no necessity to check if onClick is function, because you mentioned it in type definition. and as you marked it as optional, we should should check if this function is passed here via props before calling it.
so it is the same, but shorter))